biodatasets/scielo/scielo.py

```diff
@@ -0,0 +1,209 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Parallel corpus of full-text articles in Portuguese, English and Spanish from SciELO.
+"""
+from typing import IO, Any, Generator, List, Optional, Tuple
+
+import datasets
+from utils import schemas
+from utils.configs import BigBioConfig
+from utils.constants import Tasks
+
+_CITATION = """\
+@inproceedings{soares2018large,
+  title={A Large Parallel Corpus of Full-Text Scientific Articles},
+  author={Soares, Felipe and Moreira, Viviane and Becker, Karin},
+  booktitle={Proceedings of the Eleventh International Conference on Language Resources and Evaluation (LREC-2018)},
+  year={2018}
+}
+"""
+
+_DATASETNAME = "scielo"
+
+_DESCRIPTION = """\
+A parallel corpus of full-text scientific articles collected from Scielo database in the following languages: \
+English, Portuguese and Spanish. The corpus is sentence aligned for all language pairs, \
+as well as trilingual aligned for a small subset of sentences. Alignment was carried out using the Hunalign algorithm.
+"""
+
+_HOMEPAGE = "https://sites.google.com/view/felipe-soares/datasets#h.p_92uSCyAjWSRB"
+
+_LICENSE = "CC BY 4.0"
+
+_URLS = {
+    "en-es": "https://ndownloader.figstatic.com/files/14019287",
+    "en-pt": "https://ndownloader.figstatic.com/files/14019308",
+    "en-pt-es": "https://ndownloader.figstatic.com/files/14019293",
+}
+
+_SUPPORTED_TASKS = [Tasks.TRANSLATION]
+
+_SOURCE_VERSION = "1.0.0"
+_BIGBIO_VERSION = "1.0.0"
+
+
+class ScieloDataset(datasets.GeneratorBasedBuilder):
+    """Parallel corpus of full-text articles in Portuguese, English and Spanish from SciELO."""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    BIGBIO_VERSION = datasets.Version(_BIGBIO_VERSION)
+
+    # NOTE: bigbio_t2t schema doesn't allow only for more than two texts in text-to-text schema.
+    #  en-pt-es translation is not implemented using the bigbio schema
```

Comment:
maybe not ideal, but you could define a `source` schema and config that included all languages 
```
BigBioConfig(
            name="scielo_source",
            version=SOURCE_VERSION,
            description="scielo all languages",
            schema="source",
            subset_id="scielo",
        ),
```

Reply:
great idea, I will fix it!